src/components/hooks/use-deriv-ws.tsx

```diff
@@ -0,0 +1,10 @@
+import { DerivApiProps, useDerivApi } from './use-deriv-api'
+
+const useDerivWS = () => {
```

Comment:
@mohammad-rework What is this hook for? 🤔 
There is already an existing `useWS` and `useSubscription` hook which is typesafe.